rust/crd/src/lib.rs

```diff
@@ -363,6 +377,41 @@ impl AirflowRole {
 
         command
     }
+    fn authentication_start_commands(
+        auth_config: &AirflowClientAuthenticationDetailsResolved,
+    ) -> String {
+        let mut commands = Vec::new();
+
+        let mut tls_client_credentials = BTreeSet::new();
+
+        for auth_class_resolved in &auth_config.authentication_classes_resolved {
+            match auth_class_resolved {
+                AirflowAuthenticationClassResolved::Oidc { provider, .. } => {
+                    tls_client_credentials.insert(&provider.tls);
+
+                    // WebPKI will be handled implicitly
+                }
+                AirflowAuthenticationClassResolved::Ldap { .. } => {}
+            }
+        }
+
+        for tls in tls_client_credentials {
+            commands.push(tls.tls_ca_cert_mount_path().map(|tls_ca_cert_mount_path| {
+                Self::add_cert_to_python_certifi_command(&tls_ca_cert_mount_path)
+            }));
+        }
+
+        commands
+            .iter()
+            .flatten()
+            .cloned()
+            .collect::<Vec<_>>()
+            .join("\n")
+    }
+    // Adding certificate to the mount path for airflow startup commands
```

Comment:
**nit:** The coding style guide suggests an empty line between functions.

```suggestion

    // Adding certificate to the mount path for airflow startup commands
```

Reply:
done https://github.com/stackabletech/airflow-operator/pull/530/commits/b4d0e2fb818c5144a293d0583d4232eef9216661